tests/unittests/net/test_dhcp.py

```diff
@@ -544,16 +511,15 @@ def test_dhcp_discovery_run_in_sandbox(self, m_subp, m_kill, m_getppid):
     @mock.patch("cloudinit.net.dhcp.util.get_proc_ppid")
     @mock.patch("cloudinit.net.dhcp.os.kill")
     @mock.patch("cloudinit.net.dhcp.subp.subp")
-    def test_dhcp_discovery_outside_sandbox(self, m_subp, m_kill, m_getppid):
-        """dhcp_discovery brings up the interface and runs dhclient.
-
-        It also returns the parsed dhcp.leases file generated in the sandbox.
-        """
+    @mock.patch("cloudinit.util.wait_for_files", return_value=False)
+    def test_dhcp_discovery_setup_and_run_dhclient(
```

Comment:
I think this test is now a duplicate of the test above it. If so, let's remove it.